doc/source/usersguide/installation.rst

```diff
@@ -91,24 +96,63 @@ in PyTables, so this will be used in case it is not found in the
+versions of a library are installed, then it is possible to set the path
+to the particular resource using the environment.  The following environment
+variables can be used for the purpose:
+
+* :envvar:`HDF5_DIR`,
+* :envvar:`LZO_DIR`,
+* :envvar:`BZIP2_DIR`,
+* :envvar:`BLOSC_DIR`, or
+* :envvar:`BLOSC2_DIR`
+
+One may also specify the locations of the resource root directories on the
+`setup.py`` command line.  The following options are available:
+
+* `--hdf5`,
+* `--blosc`,
+* `--lzo`,
+* `--bzip2`, or
+* `--blosc2`
+
+ 1. the command line arguments to `setup.py` (e.g. `--blosc2`) has the
+    highest precedence
+ 2. If the command line argument for a specific library is not specified and
+    the corresponding environment variables exists (e.g. `BLOSC2_DIR`),
+    then its value is used.
+ 3. If the target library (`blosc2` in the example) can't still be found,
+    then `pkg-config` is used for further detection, if it is available.
+ 4. If the library is still not found, then a selection of hardcoded paths
+    is used
+
+Please note that, for the specific case of Blosc2, the list of search paths
+also includes the ones used by the `python-blosc2` wheel package to install
+the relevant header files and libraries.
+
 **Unix**
 
-    setup.py will detect HDF5, Blosc, LZO, or bzip2 libraries and include
-    files under :file:`/usr` or :file:`/usr/local`; this will cover most
-    manual installations as well as installations from packages.  If setup.py
-    can not find libhdf5, libhdf5 (or liblzo, or libbz2 that you may wish to
-    use) or if you have several versions of a library installed and want to
-    use a particular one, then you can set the path to the resource in the
-    environment, by setting the values of the :envvar:`HDF5_DIR`,
-    :envvar:`LZO_DIR`, :envvar:`BZIP2_DIR` or :envvar:`BLOSC_DIR` environment
-    variables to the path to the particular resource. You may also specify the
+    setup.py will detect HDF5, Blosc, Blosc2, LZO, or bzip2 libraries and
+    include files under :file:`/usr` or :file:`/usr/local`; this will cover
+    most manual installations as well as installations from packages.
+    If setup.py can not find libhdf5, libhdf5 (or liblzo, or libbz2 that
+    you may wish to use) or if you have several versions of a library
+    installed and want to use a particular one, then you can set the path
+    to the resource in the environment, by setting the values of the
+    :envvar:`HDF5_DIR`, :envvar:`LZO_DIR`, :envvar:`BZIP2_DIR`,
+    :envvar:`BLOSC_DIR` or :envvar:`BLOSC2_DIR` environment variables to
+    the path to the particular resource. You may also specify the
     locations of the resource root directories on the setup.py command line.
     For example::
 
         --hdf5=/stuff/hdf5-1.8.12
         --blosc=/stuff/blosc-1.8.1
         --lzo=/stuff/lzo-2.02
         --bzip2=/stuff/bzip2-1.0.5
+        --blosc2=/stuff/blosc2-2.0.0
```

Comment:
I think blosc2 dependency is supported only via the wheel, so I don't think we should be including lines like this.